src/main/docs/guide/gettingStarted.adoc

```diff
@@ -1,10 +1,14 @@
 To enable this support you should add the following dependencies to your build configuration:
 
 dependency:micronaut-openapi[scope="annotationProcessor", version="{version}", groupId="io.micronaut.configuration"]
+for Groovy:
+dependency:micronaut-openapi[scope="compile", version="{version}", groupId="io.micronaut.configuration"]
```

Comment:
This needs to be `compileOnly`

Reply:
Ok, may I ask why? Sice I just wanted it to start working I didn't go into details

Reply:
Because for Groovy, the dependencies in the annotation scope processor only need to be `compileOnly`.

If you create a new application with `openapi` feature using https://launch.micronaut.io, you can check it:

![image](https://user-images.githubusercontent.com/559192/86905351-3c180280-c112-11ea-998a-00223d851560.png)

After that, If you want the swagger-ui, you can add what you already submitted in this other PR: https://github.com/micronaut-projects/micronaut-openapi/pull/280/files

I've done that: Generate new project with Launch + adding the snippet in your PR, and it works as expected:

![image](https://user-images.githubusercontent.com/559192/86905540-7d101700-c112-11ea-9750-85999eddd988.png)